### ISSUE TITLE
"PWA Stats" heading should be read via voiceover

### ISSUE THREAD

**Issue by gerardo-rodriguez** [cloudfour/pwastats#44]
Currently, when using voiceover, the actual heading "PWA Stats" is not read out. Missing `aria-label` maybe?

**Comment by tylersticka** [comment]
Accidentally posted this in #43:

> It actually needs an `id` on the `<title>` and an `aria-describedby` attribute. See: https://css-tricks.com/accessible-svgs/#article-header-id-6
